static/bidder-info/360playvid.yaml

```diff
@@ -0,0 +1,22 @@
+aliasOf: "teqblaze"
+endpoint: "https://ssp.360playvid.com/pserver"
+maintainer:
+  email: "prebid@360playvid.com"
+capabilities:
+  site:
+    mediaTypes:
+      - banner
+      - video
+      - native
+  app:
+    mediaTypes:
+      - banner
+      - video
+      - native
+userSync:
+  redirect:
+    url: "https://cookie.360playvid.com/pbserver?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&ccpa={{.USPrivacy}}&gpp={{.GPP}}&gpp_sid={{.GPPSID}}&redirect={{.RedirectURL}}"
+    userMacro: "[UID]"
```

Comment:
Is redirect set up correctly? I'm getting a 204 using the following: `https://cookie.360playvid.com/pbserver?gdpr=&gdpr_consent=&ccpa=&gpp=&gpp_sid=&redirect=http%3A%2F%2Flocalhost%3A8000%2Fsetuid%3Fbidder%3D360playvid%26gdpr%3D%26gdpr_consent%3D%26gpp%3D%26gpp_sid%3D%26f%3Di%26uid%3D%5BUID%5D`

<img width="905" height="61" alt="Screenshot 2025-08-28 at 2 43 33 PM" src="https://github.com/user-attachments/assets/92ed71a4-f4ba-435e-8b0f-407f96f63900" />

Reply:
Verified redirect is now working properly hitting the setuid endpoint with the 360playvid uid stored in the PBS cookie
<img width="1028" height="88" alt="Screenshot 2025-09-04 at 11 57 27 AM" src="https://github.com/user-attachments/assets/e2377f20-31b7-4cbf-8c10-ae9d0d86aab9" />